### ISSUE TITLE
Expand/Collapse link on content component on landing page inaccessible with screen reader

### ISSUE THREAD

**Issue by jakobkolb** [jakobkolb/lizu-homepage#15]
When using a screen reader (e.g. from samsung accessibility features) the `.../mehr` option at the end of the text is invisible (inaudible) and cannot be selected. We need to make this a button or some other accessible element.
